reconcile/utils/jump_host.py

```diff
@@ -41,13 +41,29 @@ def cleanup(self):
 
 class JumpHostSSH(JumpHostBase):
     bastion_tunnel = {}
+    local_ports = []
     tunnel_lock = threading.Lock()
 
     def __init__(self, jh, settings=None):
         JumpHostBase.__init__(self, jh, settings=settings)
 
         self.known_hosts = self.get_known_hosts(jh)
         self.init_known_hosts_file()
+        self.local_port = self.get_random_port() \
+            if jh['localPort'] is None else jh['localPort']
```

Comment:
Could the `localPort` key not exist leading to a `KeyError`?  If so, should you use `jh.get('localPort')`?

Reply:
This shouldn't be empty since it is initialized with other jh values, but imo safer is always better.